### DISCUSSION TITLE
Make it easier to navigate to organization

### DISCUSSION THREAD

**Discussion by nychen** [community/community#121850] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It seems like the homepage has shortcuts to everything except for my organizations. Would love for one-click access to an organization from my account homepage.

**Comment by KeyboardRage** [comment]
I too would find it very useful to have easy access to organizations from the homepage.

I use organizations at work and in private, so I navigate to them a lot, multiple times a day too when I need to work with multiple repos within a single organization. So it is always such a hassle as it often requires three steps:
- open profile in the top right
- click "my profile" or "my organizations"
- click the desired organization

**Comment by eolivares-e** [comment]
this is so useful, everyday I'm like 5-7 clicks away from going to desired organization repos and the switch dashboard content is not helping neither.
